### PR TITLE
Issue #205 - Use tidy verse

### PR DESCRIPTION
**Pull request by rsangole**
Changes full/global.dcf. Replaces packages with tidyverse. Reshape with reshape2.

### CODE REVIEW

File: inst/defaults/full/config/global.dcf [hunk 1]
```diff
@@ -8,7 +8,7 @@ munging: TRUE
 logging: FALSE
 logging_level: INFO
 load_libraries: FALSE
-libraries: reshape, plyr, dplyr, ggplot2, stringr, lubridate
+libraries: reshape2, tidyverse, plyr, lubridate
```
Comment (Hugovdberg) [inline inst/defaults/full/config/global.dcf]: It is not recommended to load `plyr` after `dplyr` and raises a warning, please swap the two arround. Also, `stringr` is installed as a dependency for `tidyverse` but is not actually loaded automatically, so should not be removed.

Comment (rsangole) [reply inst/defaults/full/config/global.dcf]: Thanks for the feedback. Making these changes.

Comment (rsangole) [reply inst/defaults/full/config/global.dcf]: I'm also tempted to insert `forcats` in there. Thoughts? It's been extremely useful in re-coding factors, grouping factors etc.

**Comment by Hugovdberg** [comment]
The new commit looks reasonable. We do need to think about how we communicate we change to `reshape2`, since it isn't a 100% drop in replacement for `reshape`. `migrate.project` doesn't touch the configuration so that's no issue, but perhaps we can notify users via some other way.
Also, @KentonWhite should the suggested packages in `DESCRIPTION` be updated as well, or are those only for packages used internally in `ProjectTemplate`?

**Comment by rsangole** [comment]
That's a good point. How is this type of communication typically handled for a package update?

**Comment by KentonWhite** [comment]
I convinced myself we don't need to worry.  Legacy projects will be written to use the `reshape` package. In those projects they will have a `global.dcf` file with the `reshape` package not the `reshape2` package.  Only new projects will be created with the `reshape2` package.  By definition they do not have existing code and so nothing to break!  If they want to use `reshape` with a new project they can manually change the config file.

I think it is fine to make a note of the change in the `NEWS.md` file.

**Comment by rsangole** [comment]
Perfect. I've put in reshape2 in the pull request.

Thoughts on also including `forcats`? It's something I use everyday, but not sure about you guys.

**Pull request merged**